src/api/volume.ts

```diff
@@ -1,11 +1,13 @@
-import { useQuery } from "@tanstack/react-query"
+import { useQueries } from "@tanstack/react-query"
 import { addDays } from "date-fns"
 import { gql, request } from "graphql-request"
-import { Maybe, undefinedNoop } from "utils/helpers"
+import { Maybe, normalizeId, undefinedNoop } from "utils/helpers"
 import { QUERY_KEYS } from "utils/queryKeys"
 import { u32 } from "@polkadot/types-codec"
+import BN from "bignumber.js"
+import { BN_0 } from "../utils/constants"
 
-export const getTradeVolume = (assetId: u32) => async () => {
+export const getTradeVolume = async (assetId: u32) => {
```

Comment:
This might be unnecessary? If we return the async function, we don't need to do the thing seen in `useTradeVolumes`